thirdeye/thirdeye-frontend/app/helpers/utils.js

```diff
@@ -488,6 +488,21 @@ export function isInverse(urn, entities) {
   return (entities[urn].attributes.inverse[0] === 'true');
 }
 
+/**
+ * Extracts information about whether a metric is tagged as additive or not.
+ *
+ * @param {string} urn metric urn
+ * @param {Object} entities entities cache
+ * @returns {boolean} if metric is tagged as additive
+ */
+export function isAdditive(urn, entities) {
+  if (!entities) { return false; }
```

Comment:
perhaps a try catch would be more succinct here. I feel it's fine to use for deeply nested objects where a null can pop up at any level.
```
try {
  return (entities[urn].attributes.additive[0] === 'true')
} catch (error) {
  return false; 
}
```
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Control_flow_and_error_handling#try...catch_statement

Reply:
thanks. fixed